Doberman/AlarmMonitor.py

```diff
@@ -37,6 +38,43 @@ def get_connection_details(self, which):
             self.logger.critical('Could not load connection details for %s' % which)
             return None
 
+    def send_phonecall(self, phone_numbers, message):
+        # Get connection details
+        connection_details = self.get_connection_details('twilio')
+        if connection_details is None:
+            raise KeyError("No connection details obtained from database.")
+        # Compose connection details and addresses
+        url = connection_details['url']
+        fromnumber = connection_details['fromnumber']
+        auth = tuple(connection_details['auth'])
+        maxmessagelength = int(connection_details['maxmessagelength'])
+
+        if not phone_numbers:
+            raise ValueError("No phone number given.")
+
+        message = str(message)
+        # Long messages are shortened to avoid excessive fees
+        if len(message) > maxmessagelength:
+            message = ' '.join(message[:maxmessagelength+1].split(' ')[0:-1])
+            message = '<p>' + message + '</p>'
+            message += '<p>Message shortened.</p>'
+            self.logger.warning(f"Message exceeds {maxmessagelength} "
+                                "characters. Message will be shortened.")
+        message = f"This is the {self.db.experiment_name} alarm system. " + message
+        if len(phone_numbers) == 1:
+            phone_numbers = [phone_numbers]
+        for tonumber in phone_numbers:
+            data = {
+                'To': '+491719561419',
+                'From': '+4915223268200',
```

Comment:
Do we want to leave phone numbers in plain text like this in a file we just upload to the open internet? This looks like some scaffolding that got forgotten.

Reply:
Yes. Gone. Dunno if there is a way to remove them properly though.